packages/test/test-end-to-end-tests/src/test/SummarizerWithSearch.spec.ts

```diff
@@ -116,12 +115,17 @@ async function loadSummarizer(
 		);
 	});
 
-	const defaultDataStore = await requestFluidObject<TestDataObject1>(
-		summarizerContainer,
-		"default",
-	);
+	// entryPoint of a summarizer container is the Summarizer object
+	const summarizer = await summarizerContainer.getEntryPoint();
+	// The runtime prop is private to the Summarizer class
+	const containerRuntime = (summarizer as any).runtime as ContainerRuntime;
```

Comment:
Example of `(summarizer as any).runtime`.
Note: this is done in a number of places outside of these changes as well

Reply:
You could do this as a follow-up too since you mentioned there are other places this pattern is used.

Reply:
`ContainerRuntime` has an override directly in the constructor that sets the `entryPoint` to be the `Summarizer` object when we are in a summarizer container.
https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/containerRuntime.ts#L1733